hack/postgres-passwords-migration.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+# This script must be run to migrate all database passwords
+# before upgrading the synopsys-operator
+
+function usage(){
+  echo -e "usage:      ${0} [NAMESPACE1] [NAMESPACE2] [...]\n"
+  echo -e "NAMESPACE:  The blackduck namespace to migrate the secret from\n"
+}
+
+# check for number of arguments
+[ "$#" -lt 1 ] && echo -e "ERROR: At least one NAMESPACE must be provided.\n" && usage && exit
+
+for namespace
+do
+  # ADMIN_PASSWORD
+  kubectl patch blackduck $namespace --type merge -p '{"spec": {"adminPassword" : "'$(kubectl get secret -n synopsys-operator blackduck-secret -o jsonpath='{.data.ADMIN_PASSWORD}')'"}}'
+  # USER_PASSWORD
+  kubectl patch blackduck $namespace --type merge -p '{"spec": {"userPassword" : "'$(kubectl get secret -n synopsys-operator blackduck-secret -o jsonpath='{.data.USER_PASSWORD}')'"}}'
```

Comment:
get the password values once and then have a single patch command to update all the passwords in the blackduck resource..

Reply:
Done.